cmds/core/kexec/kexec_linux.go

```diff
@@ -40,13 +42,15 @@ type options struct {
 	load         bool
 	exec         bool
 	debug        bool
+	extra        string
 	modules      []string
 }
 
 func registerFlags() *options {
 	o := &options{}
 	flag.StringVarP(&o.cmdline, "cmdline", "c", "", "Append to the kernel command line")
 	flag.StringVar(&o.cmdline, "append", "", "Append to the kernel command line")
+	flag.StringVarP(&o.extra, "extra", "x", "", "Add a cpio containing extra files")
```

Comment:
typo ? looks like need be empty - per example usage: kexec -extra 'kmsg hosts' -i initrd.cpio kernel

Reply:
it is empty. This is StringVarP, not StringVar.

Reply:
my bad.. occasionally asking the obvious unnecessarily